src/main/java/edu/harvard/iq/dataverse/flyway/StartupFlywayMigrator.java

```diff
@@ -20,6 +20,7 @@ void migrateDatabase() {
 
         Flyway flyway = Flyway.configure()
                 .dataSource(dataSource)
+                .validateOnMigrate(false)
```

Comment:
We probably wouldn't want lack of validation in production.

Reply:
If we turn on validation then we would have to either:
1) Manually fix checksum field in flyway table on server
2) Use mvn flyway:repair (not sure what will actually happen)
3) Not to edit existing migration but write a completely new one - it would be rather difficult; it would have to contain inserts on some condition (to assure that it will not break existing db)

Reply:
What do you think?